python/spire/modelChecker/modelChecker_commands.py

```diff
@@ -15,6 +15,17 @@ def trailingNumbers(nodes, SLMesh):
     return trailingNumbers
 
 
+def transformNames(nodes, SLMesh):
```

Comment:
nit. transformNames is kind of ambiguous. a more descriptive name with docstring what it's looking for would be good.